project_stage/static/src/less/kanban_dashboard.less

```diff
@@ -0,0 +1,5 @@
+// ------- Kanban dashboard variables -------
+
+.o_kanban_dashboard_project {
```

Comment:
Note that Odoo 13 doesn't use `less` styles, but `scss`.

This style breaks the entire backend asset generation. You can probably reproduce this issue by regenerating the assets bundle in your development environment.

Reply:
Seems like this can be removed outright.